### ISSUE TITLE
Dropdown input duplicates options when assigned async function to options prop and that function returns options synchronously.

### ISSUE THREAD

**Issue by KoenCa** [formkit/formkit#975]
### Reproduction

https://formkit.link/3e4c49614794b5b42730ce7641cd06e9

### Describe the bug

Dropdown options are duplicated in a form with a v-model with default values.
- When I leave the object empty, then options aren't duplicated
- When I add the `always-load-on-open` prop, then it also does work
- When I change the type to `taglist` or `autocomplete` then it also does work

### Environment

• OS: MacOS
• Browser: Chrome
• Version 1.2

**Comment by justin-schroeder** [comment]
Yikes — we’ll look at this today, thank you for the report.

**Comment by sashamilenkovic** [comment]
@KoenCa Hey! I looked into this, good find! Looks like setting your options function to async while returning synchronously is causing a race condition issue. Should have a fix out for this a little later.

**Comment by sashamilenkovic** [comment]
@KoenCa This is fixed at [0.117.0](https://www.npmjs.com/package/@formkit/pro/v/0.117.0)

**Comment by KoenCa** [comment]
@sashamilenkovic Thanks for the quick fix! I was implementing this in my work project, but in one dropdown it can still happen so I tried to reproduce it in the playground again and I managed to do it. Now it only happens when I add `load-on-created` prop to the dropdown.

**Bit more complex example:**
https://formkit.link/4cd8a4edd33169c7907cb555bf67016e

**Simple example**
https://formkit.link/c566a33ecc700c76894120d94c82c152
